util/config/compileline

```diff
@@ -37,6 +46,15 @@ $make = "$make --no-print-directory";
 
 sub mysystem {
   my $cmd = shift;
+
+  # Uncomment for debug output
```

Comment:
Would this be used frequently enough to control by a flag rather than uncommenting the code?